samples/dungeon/common/src/main/java/com/squareup/sample/dungeon/PlayerWorkflow.kt

```diff
@@ -35,16 +35,16 @@ class PlayerWorkflow(
   private val cellsPerSecond: Float = 15f
 ) : StatefulWorkflow<ActorProps, Movement, Nothing, Rendering>() {
 
-  sealed class Action : WorkflowAction<ActorProps, Movement, Nothing> {
+  sealed class Action : WorkflowAction<ActorProps, Movement, Nothing>() {
 
     class StartMoving(private val direction: Direction) : Action() {
-      override fun Updater<ActorProps, Movement, Nothing>.apply() {
+      override fun Updater.apply() {
```

Comment:
This is pretty nice. Makes this pattern that much more tolerable. And it does get more use than you might think, especially with teams that are better balanced btw. Android and iOS